src/main/java/org/jenkinsci/plugins/workflow/cps/SnippetizerLink.java

```diff
@@ -196,6 +196,22 @@ public boolean inNewWindow() {
     }
 
     @Extension(ordinal = 600L)
+    public static class ExamplesLink extends SnippetizerLink {
+
+        @Nonnull
+        @Override
+        public String getUrl() {
+            return "https://github.com/jenkinsci/pipeline-examples";
```

Comment:
That links to the github.com repository.  I believe it should link to the jenkins.io documentation page rather than the github repository.

```suggestion
            return "https://jenkins.io/doc/pipeline/examples/";
```

Reply:
That's a better offer.